### ISSUE TITLE
No mapping file found named 'Order.orm.xml'

### ISSUE THREAD

**Issue by muhammadhafeez** [Sylius/Sylius#8098]
I'm trying to move site from local to live but getting this error. I've been facing this error since last 2 days but didn't find any fix. Please help 

`
Fatal error: Uncaught exception 'Doctrine\Common\Persistence\Mapping\MappingException' with message 'No mapping file found named 'Order.orm.xml' for class 'Sylius\Component\Core\Model\Order'.' in /home3/webi/public_html/sylius.jobux.pk/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:75 Stack trace: #0 /home3/webi/public_html/sylius.jobux.pk/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/Driver/SymfonyFileLocator.php(237): Doctrine\Common\Persistence\Mapping\MappingException::mappingFileNotFound('Sylius\\Componen...', 'Order.orm.xml') #1 /home3/webi/public_html/sylius.jobux.pk/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/Driver/FileDriver.php(115): Doctrine\Common\Persistence\Mapping\Driver\SymfonyFileLocator->findMappingFile('Sylius\\Componen...') #2 /home3/webi/public_html/sylius.jobux.pk/vendor/doctrine/orm/lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php(58): Doctrine\Common\Persistence\Mapping\Driver\FileDriver->getElement('Sylius\\Componen...') #3 /ho in /home3/webi/public_html/sylius.jobux.pk/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php on line 75`

Tried two difference hosting but same error.

**Comment by pjedrzejewski** [comment]
Can you tell us more about the hosting you use? Also, does it work for you locally?

**Comment by muhammadhafeez** [comment]
@pjedrzejewski Thanks for your reply. This error pop-out when we try to change environment from dev to prod.

We're using two different Linux hosting but same error on both. 

Here are steps. 
1. I did install Sylius on local machine (window's 7).
2. Then i zip everything and uploaded to Amazon EC2 Linux (ubuntu).
3. I try to access http://13.113.175.122/web/ but a blank page. 
4. I checked prod.log file and got an error that mention above. 

Dev mod is working fine  here
http://13.113.175.122/web/app_dev.php/en_US/

For more info about environment  http://13.113.175.122/info.php
